Astro Example could not run

**Describe the bug**

Astro example doesnt run, i was looking forward to trying out `better-auth` but instead was faced with debugging `pnpm`, a tool ive never used before, when it failed to find the local dependencies. All part of a beta release I suppose, best of luck with the launch!

**To Reproduce**
Steps to reproduce the behavior:

Running on Ubuntu 24.04.

1. clone the repo
2. install pnpm
3. `cd examples/astro-example`
4. run `pnpm install`
5. run `pnpm run dev`

```
> pnpm better-auth migrate
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "better-auth" not found
```
6. run `pnpm astro dev`

```
10:38:29 [ERROR] [vite] Error: The following dependencies are imported but could not be resolved:

  better-auth/client/plugins (imported by /home/pete/me/better-auth/examples/astro-example/src/libs/auth-client.ts)
  better-auth/client (imported by /home/pete/me/better-auth/examples/astro-example/src/libs/auth-client.ts)
  better-auth/solid (imported by /home/pete/me/better-auth/examples/astro-example/src/libs/auth-client.ts)

Are they installed?
```

--- a/examples/astro-example/package.json
+++ b/examples/astro-example/package.json
@@ -5,7 +5,7 @@
   "version": "0.0.1",
   "scripts": {
     "dev": "pnpm migrate:auth && astro dev --port 3000",
-    "migrate:auth": "pnpm better-auth migrate",
+    "migrate:auth": "pnpx @better-auth/cli migrate",
     "start": "astro dev",
     "build": "astro check && astro build",
     "preview": "astro preview",

--- a/examples/astro-example/src/auth.ts
+++ b/examples/astro-example/src/auth.ts
@@ -1,9 +1,6 @@
 import { betterAuth } from "better-auth";
 import { passkey, twoFactor } from "better-auth/plugins";
 import Database from "better-sqlite3";
-import { Resend } from "resend";
-
-const resend = new Resend(import.meta.env.RESEND_API_KEY);
 
 export const auth = betterAuth({
 	database: new Database("./db.sqlite"),
@@ -18,8 +15,12 @@ export const auth = betterAuth({
 	},
 	socialProviders: {
 		google: {
-			clientId: import.meta.env.GOOGLE_CLIENT_ID || "",
-			clientSecret: import.meta.env.GOOGLE_CLIENT_SECRET || "",
+			clientId: import.meta.env.GOOGLE_CLIENT_ID!,
+			clientSecret: import.meta.env.GOOGLE_CLIENT_SECRET!,
+		},
+		github: {
+			clientId: import.meta.env.GITHUB_CLIENT_ID!,
+			clientSecret: import.meta.env.GITHUB_CLIENT_SECRET!,
 		},
 	},
 	plugins: [

